app/screens/channel_info/channel_info_header.js

```diff
@@ -172,6 +177,29 @@ export default class ChannelInfoHeader extends React.PureComponent {
                         {displayName}
                     </Text>
                 </View>
+                {isCustomStatusEnabled && type === General.DM_CHANNEL && customStatus.emoji &&
+                    <View
+                        style={[style.row, style.customStatusContainer]}
+                        testID={`${testID}.custom_status`}
+                    >
+                        <Text
+                            style={style.iconContainer}
+                            testID={`custom_status.emoji.${customStatus.emoji}`}
+                        >
+                            <Emoji
+                                emojiName={customStatus.emoji}
+                                size={20}
+                            />
+                        </Text>
+                        <Text
+                            style={style.customStatusText}
+                            ellipsizeMode='tail'
+                            numberOfLines={1}
+                        >
+                            {customStatus.text}
+                        </Text>
```

Comment:
Why not CustomStatusText?

Reply:
CustomStatusText component is designed to wrap long texts in multiple lines which does not match the behaviour we want here.

Reply:
CustomStatusText could receive props fo nunberOfLines and ellipsisMode